stack/rush-stack-compiler/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "./includes/tsconfig-node.json",
+
+  "compilerOptions": {
+    "outDir": "lib",
+
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "experimentalDecorators": true,
+    "strictNullChecks": true,
+    "types": []
+  },
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.tsx"
+  ],
+  "exclude": [
+    "node_modules",
+    "lib"
+  ]
```

Comment:
?

Reply:
If we have a cyclic dependency on  "@microsoft/node-library-build", perhaps we could get the tsconfig.json that way.

If a project is depending on **rush-stack** (possibly indirectly via a rig), I think our design is that the project should /always/ also depend on something that gives it **rush-stack-compiler**.  If not **rush-stack-compiler** itself, then some wrapper that adds custom rules and config.

If you did this here, then the tsconfig-node.json paths would work out correctly.

---
In reply to: [228674633](https://github.com/Microsoft/web-build-tools/pull/905#discussion_r228674633) [](ancestors = 228674633)